nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs

```diff
@@ -116,8 +118,14 @@ protected void Awake()
                 {
                     _elemental = filter;
                     SortedData(_grade, _elemental);
+                    AudioController.PlayClick();
```

Comment:
It is recommended to place this line in the input event method.

Reply:
This line already placed in onValueChanged event method.